src/component/header/header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import iconFile from "../../icons-file.svg";
+import styles from "./header.module.scss";
+
+const Header = ({ header }) => {
+  return (
+    <header className={styles.header}>
+      <div className={styles.header__title}>
+        <div
+          className={styles.header__image}
+          style={{ backgroundImage: `url(${iconFile})` }}
```

Comment:
We should add the function to the icon at the top right.
When it is click, hide the content below it, and change the icon to up trianggle.
When it is click again, show the content below it and change the icon to down trianggle.